### DISCUSSION TITLE
Max size of tiered storage

### DISCUSSION THREAD

**Discussion by yossisadot** [microsoft/garnet#464] [Q&A]
When running with `--storage-tier true`, is there a way to limit the max size of the used storage (after which some eviction/compaction takes place)?

**Comment by badrishc** [comment]
You can set `--compaction-max-segments` to control how many segments (files) are retained on disk. The size of each segment is configured using `--segment` and is 1gb by default. The type of compaction (truncation or copy live records to tail) is configured using `--compaction-type`. Compaction will actually delete the physical files only after a checkpoint, if you do not need checkpoints, you can use the compaction type of `ShiftForced`. More details at https://microsoft.github.io/garnet/docs/getting-started/configuration.
